tools/dart_tools/bin/build_jsoncpp.dart

```diff
@@ -57,12 +57,19 @@ Future<void> buildLibrary(String buildDirectory, {bool debug}) async {
   final arguments = [
     'msbuild',
     'lib_json.vcxproj',
+    '/p:Platform=x64',
   ];
+  if (Platform.environment['APPVEYOR'] != 'True') {
+    arguments.insert(0, 'vcvars64.bat 1> nul &&');
```

Comment:
> Now, Visual C++ tools (cl.exe, link.exe and others) should be available in the PATH and required variables for CL environment and LINK enviornment set properly for command-line builds.

The tools the toolchain uses are already in the path, however `vcvars64.bat` is not so it will fail with it added (and is unnessecary with the tools in the path).